apps/app/pages/[workspaceSlug]/projects/[projectId]/cycles/[cycleId].tsx

```diff
@@ -78,6 +80,40 @@ const SingleCycle: React.FC<UserAuth> = (props) => {
       : null
   );
 
+  const { data: currentAndUpcomingCycles } = useSWR(
```

Comment:
You are making these two api requests just to check the status of current cycle?